tests/ossfuzz/fuzz_decode_stream.c

```diff
@@ -1,10 +1,13 @@
 ///////////////////////////////////////////////////////////////////////////////
 //
-/// \file       fuzz.c
+/// \file       fuzz_decode_stream.c
 /// \brief      Fuzz test program for liblzma
 //
 //  Author:     Lasse Collin
 //
+//  Original fuzz.c fuzzer, renamed to fuzz_decode_stream to conform with new
```

Comment:
There isn't a need to mention the previous filename. This information is capture in the commit history